test/toDoControllerSpec.spec.js

```diff
@@ -0,0 +1,83 @@
+describe('toDoController', function() {
+  beforeEach(module('toDoList'));
+
+  var scope, ctrl
+
+  beforeEach(inject(function($rootScope, $controller){
+    scope = $rootScope.$new();
+    ctrl = $controller('toDoController', {
+      $scope: scope
+    });
+  }));
+
+  describe('list of tasks', function() {
+    var task = [
+      {
+        "task": "get some milk",
+        "done": false
+      }
+    ]
+
+    it('starts with an empty task entry field.', function() {
+      expect(scope.enterTask).toBeUndefined();
+    });
+
+    it('can add one task item.', function() {
+      setUpHelper();
+      expect(scope.allTasks[0].task).toEqual(task[0].task);
+    });
+
+    it('are totaled and displayed.', function(){
```

Comment:
You are not testing display logic here - just that the total is correct.  Always keep in mind which layer of the software you are testing.